### PR TITLE
some NLB additions.

### PR DESCRIPTION
**Pull request by InTheCloudDan**

### CODE REVIEW

**Comment by bgdnlp** [comment]
Not directly related to this PR, but I suggest a complete rewrite of this section, splitting capabilities by load balancer type. Something in the form of:

**Classic Load Balancers** (CLB) a.k.a. **Elastic Load Balancers** (ELB):
* are used for
* can do this
* have this characteristic

**Application Load Balancers** (ALB):
* are used for
* can do this
* have this characteristic

**Network Load Balancers** (NLB):
* are used for
* can do this
* have this characteristic

Would it be useful? Maybe after merging this PR?

**Comment by QuinnyPig** [comment]
This one looks good. One more signoff, and we're set. I like your rewrite idea too.

File: README.md [hunk 1]
```diff
@@ -1021,14 +1021,14 @@ Load Balancers
...
+-	CLBs have been around since 2009, ALBs in 2016, NLBs were added in 2017 to AWS.
+-	CLBs support TCP and HTTP load balancing. ALBs support HTTP load balancing only. NLBs support TCP layer 4 load balancing.
+-	CLBs and ALBs can optionally handle termination for a single SSL certificate.
+-	All can optionally perform active health checks of instances and remove them from the destination pool if they become unhealthy.
+-	CLBs don't support complex / rule-based routing. ALBs support a (currently small) set of rule-based routing features. NLBs have most extensive routing options.
+-	CLBs can only forward traffic to a single globally configured port on destination instances, while ALBs can forward to ports that are configured on a per-instance basis, better supporting routing to services on shared clusters with dynamic port assignment (like ECS or Mesos). NLBS support multiple ports on same IP; registering targets by IP address, including targets outside the VPC for the load balancer; ECS can select unused port for scheduling a task then register a target group using this port.
```
Comment (benkehoe) [inline README.md]: Tiny note here: the "s" in "NLBs" is capitalized here, making it look like the acronym "NLBS", which confused me at first.

**Comment by InTheCloudDan** [comment]
@benkehoe thanks for the catch! Updated it to be NLBs

**Pull request merged**